lib/greed/dice.rb

```diff
@@ -0,0 +1,10 @@
+module Greed
+  class Dice
+    attr_reader :values
+
+    # Roll -num- six sided dice and place results in an array
+    def roll(num)
+      @values = (1..num).map { rand(1..6) }
```

Comment:
It's a little weird to use a map here since you're not using the things you're iterating  over, maybe something like:
`Array.new(num) { rand(1..6) }`